doc/configuration.md

```diff
@@ -3,6 +3,7 @@
 Here is the full configuration with default values:
 
 ```yaml
+# config/packages/framework.yaml
```

Comment:
I think it should be `config/config_test.yaml` or `config/packages/test/config.yaml` so that the config only exists in the `test` environment.

Reply:
I agree. Then better to name like other config 
config/packages/test/liip_fixtures.yaml